packages/dataviews/src/components/dataviews-view-config/style.scss

```diff
@@ -1,11 +1,14 @@
 .dataviews-view-config {
-	width: 320px;
-	/* stylelint-disable-next-line property-no-unknown -- the linter needs to be updated to accepted the container-type property */
-	container-type: inline-size;
-	padding: $grid-unit-20;
-	font-size: $default-font-size;
-	line-height: $default-line-height;
+	.components-popover__content {
```

Comment:
Using internal component classnames can really hinder our ability to make changes to the components without causing unintended breaking changes.

In this case, we can use the `DropdownContentWrapper` component, which is used specifically to set custom padding for dropdown content. I will open a separate PR to make the change

Reply:
https://github.com/WordPress/gutenberg/pull/65373

Reply:
Thank you for your follow up! I wasn't aware of `DropdownContentWrapper`.